zerver/lib/github.py

```diff
@@ -61,3 +66,44 @@ def get_latest_github_release_download_link_for_platform(platform: str) -> str:
         if verify_release_download_link(link):
             return link
     return "https://github.com/zulip/zulip-desktop/releases/latest"
+
+
+def get_issue_or_pr_cache_key(
+    owner: str, repo: str, type: str, issue_number: str
+) -> str:  # nocoverage
+    return f"github_issue_or_pr_url:{owner}/{repo}/{type}/{issue_number}"
```

Comment:
Please add some tests that cover this code path. You can do it nicely using the `responses` module to configure what HTTP responses GitHub is expected to return from a given API endpoint; this is important for the tests to work offline and without a real API key.

(Often we setup a fixture)